agent/exploits/cve_2024_33544.py

```diff
@@ -1,30 +1,37 @@
-"""Agent Asteroid implementation for CVE-2024-29269"""
+"""Agent Asteroid implementation for CVE-2024-33544"""
 
 import re
 
 from agent import definitions
 from agent import exploits_registry
 from agent.exploits import webexploit
 
-VULNERABILITY_TITLE = "SQL Injection in AA-Team WZone "
+VULNERABLE_VERSIONS = [
+    r"1[0-3](\.\d+){0,2}",  # Matches versions from 1.0 to 13.x.x
```

Comment:
did you test this on something like 1.0 ?

Reply:
Yeah, it worked.

Reply:
can you add unit test for it please, i see only test for second regex